minerl/herobraine/hero/handlers/agent/actions/chat.py

```diff
@@ -0,0 +1,46 @@
+__author__ = "Sander Schulhoff"
+__email__ = "sanderschulhoff@gmail.com"
+
+from minerl.herobraine.hero.handlers.agent.action import Action
+import minerl.herobraine.hero.spaces as spaces
+
+# TODO add more command support (things like what commands are allowed) from Malmo
+# TODO ensure both agents can send chats, not just first agent (again, check Malmo) 
```

Comment:
"Both" should probably mean "other"? :)

Also could this be related to Minecraft op-ness where only first agent has op powers (i.e. they can call commands like `tp`)?

Reply:
yep, changed "Both" to "other"

Interesting idea. I don't believe that to be the case.